akkaclusterfront/src/main/resources/application.conf

```diff
@@ -19,10 +19,11 @@ akka {
 
   cluster {
     auto-down-unreachable-after = 10s
-    min-nr-of-members = 2
+    //min-nr-of-members = 2
     role {
-      frontend.min-nr-of-members = 1
-      backend.min-nr-of-members = 1
+      //frontend.min-nr-of-members = 1
+      //backend.min-nr-of-members = 1
     }
+    seed-nodes = ["akka.tcp://AkkaConductRExamplesClusterSystem@127.0.0.1:8088"]
```

Comment:
You can keep seed-node configuration here - it will be overwritten `ClusterProperties.initialize` if necessary.